DatadogCore/Sources/Datadog.swift

```diff
@@ -150,7 +158,8 @@ public struct Datadog {
             uploadFrequency: UploadFrequency = .average,
             proxyConfiguration: [AnyHashable: Any]? = nil,
             encryption: DataEncryption? = nil,
-            serverDateProvider: ServerDateProvider? = nil
+            serverDateProvider: ServerDateProvider? = nil,
+            backgroundTasksEnabled: Bool = false
```

Comment:
change-request/ This is new public API and we must document it as part of `init()` doc.